### DISCUSSION TITLE
What should I translate at the "few" section on transifex?

### DISCUSSION THREAD

**Discussion by superscratch444tre** [ScratchAddons/ScratchAddons#1070] [Translation] [answered]
Hi! I am translating Scratch Addons to Romanian, and I have a question about the translation.

<img width="323" alt="few" src="https://user-images.githubusercontent.com/69812464/102597180-2de2ff00-4112-11eb-9912-71c0dedf41af.png">

At the "# sprites" and "# scripts" translations, I can see a "few" section. What should I translate there?

**Comment by WorldLanguages** [comment, marked as answer]
Well, I don't know Romanian, but some languages don't only have singular and plural. Transifex believes Romanian has singular (1 of a thing), "few" (3-5 of a thing, I guess?) and "other" (6 or more of a thing). If that's the case you can provide different translations depending on the amount of scripts (one script, few scripts, many scripts). If it's not the case, then make the "few" translation the same as "other", in other words, both would just be plural.

**Reply by superscratch444tre** [reply]
Ok, Romanian has a kind of "few" plural (2-19 of a thing), so I'll translate "# scripts", assuming that # is a number between 2 and 19, right?

**Reply by WorldLanguages** [reply]
@superscratch444tre exactly.
